float64 increment
---
geometry_msgs/PointStamped[] points
